fitsnap3/__init__.py

```diff
@@ -73,6 +73,12 @@
 except Exception as e:
     print("Trouble importing natsort package, exiting...")
     raise e
+#try:
+#    import lammps
+#    print("LAMMPS version: ",lammps.version())
+#except Exception as e:
+#    print("Trouble importing LAMMPS library, exiting...")
+#    raise e
```

Comment:
Is it necessary to remove this check for lammps.py at this location? If not, I would leave it in.

Reply:
I would like to have the version check here in __init__ for debugging purposes, even if there is a lammps exception raised later in deploy.py if lammps cannot be loaded. The problem is, that in contrast to the lammps doc pages, lammps has no attribute version(). I left the code here as a reminder to track this down later.

Reply:
>>> lmp.version()
20191120